editor/register_editor_types.cpp

```diff
@@ -272,6 +272,8 @@ void register_editor_types() {
 	GLOBAL_DEF("editor/import/reimport_missing_imported_files", true);
 	GLOBAL_DEF("editor/import/use_multiple_threads", true);
 
+	GLOBAL_DEF(PropertyInfo(Variant::INT, "editor/import/atlas_max_width", PROPERTY_HINT_RANGE, "1,8196,1,or_greater"), 2048);
```

Comment:
Shouldn't it be `8192 = 2^13`?
```suggestion
	GLOBAL_DEF(PropertyInfo(Variant::INT, "editor/import/atlas_max_width", PROPERTY_HINT_RANGE, "1,8192,1,or_greater"), 2048);
```

Reply:
True, thank you, got the powers of two confused